hw/ip/pwm/pwm.core

```diff
@@ -0,0 +1,65 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:ip:pwm:0.1"
+description: "pwm"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:constants:top_pkg
```

Comment:
I don't see the usage of top_pkg in pwm. Could you please remove?

Reply:
Will do, no problem